tests/test_utils.py

```diff
@@ -53,5 +53,13 @@ def test_calculate_journal_volume(self):
     def test_get_last_commit_to_master(self):
         self.assertIsNotNone(utils.get_last_commit_to_master())
 
+    def test_text_from_affiliation_elements(self):
```

Comment:
These kind of tests 'could' be parametrised, it would make it a bit less repetitive and you could write more cases easily.

With pytest: [I / we use](https://docs.pytest.org/en/latest/parametrize.html)
or something like [parameterized](https://github.com/wolever/parameterized) (I know some of the older projects also use [ddt](https://github.com/txels/ddt) as well)

However, I know it's not worth installing another package or changing test framework to accommodate for this instance.

Reply:
I didn't see ddt used yet so I didn't introduce it. I think I may add it now though, it is reason enough.